misc/testerina/modules/testerina-core/src/main/ballerina/annotation_processor.bal

```diff
@@ -144,8 +173,8 @@ function hasTest(string name) returns boolean {
                 if (filter.includes(WILDCARD)) {
                     boolean|error wildCardMatch = matchWildcard(testName, filter);
                     if (wildCardMatch is boolean && wildCardMatch && matchModuleName(filter)) {
```

Comment:
```suggestion
                    return (wildCardMatch is boolean && wildCardMatch && matchModuleName(filter));
```

Reply:
Let's introduce a variable and return the value. The condition is not needed right?